opflexagent/config.py

```diff
@@ -30,6 +30,16 @@
                 default=['*'],
                 help=_("List of the physical networks managed by this agent. "
                        "Use * for binding any opflex network to this agent")),
+    cfg.ListOpt('firewall_map',
```

Comment:
this firewall map looks a lot like a list of driver entry points. I would suggest we follow that kind of implementation that is widely used in openstack... Looking at how ML2 works with mechanism drivers is a good start! We can have a webex around this